File: LeetCode/Simulation/JS/Leetcode1619-mean-of-array-after-removing-some-elements.js
var trimMean = function(arr) {
    arr.sort((a,b) => a - b);
    const n = arr.length;
    let acc = 0;
    for(let i = Math.floor(n / 20);i < Math.floor((n * 19) / 20);i++) {
        acc += arr[i];
    }
    return acc / (0.9 * n);
};

File: LeetCode/Simulation/Golang/Leetcode1619-mean-of-array-after-removing-some-elements.go
package sbgo

func trimMean(arr []int) float64 {
    n,acc := len(arr),0
    sort.Ints(arr)
    for i := n / 20;i < (19 * n) / 20;i++ {
        acc += arr[i]
    }
    return float64(acc) / (0.9 * float64(n))
}
